### PR TITLE
Diff of backport to the original WCM Core code

### PR DESCRIPTION
**Pull request by HitmanInWis**
This PR is simply to allow review of the changes to the WCM Core Components code that was pulled into the backport.

### CODE REVIEW

File: core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/form/ContainerImpl.java [hunk 1]
```diff
@@ -129,10 +111,6 @@ private void initModel() {
                 redirect = contextPath + redirect;
             }
         }
-
-        if (!StringUtils.equals(request.getRequestPathInfo().getExtension(), ExporterConstants.SLING_MODEL_EXTENSION)) {
```
Comment (HitmanInWis) [inline core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/form/ContainerImpl.java:133]: I think I accidentally removed this thinking it had to do with sling model exporting, but it looks like this code is supposed to run if *not* doing sling model export, so I wonder if we should restore `runActionTypeInit(formStructureHelper);` without the `if` statement

File: core/src/main/java/com/day/cq/wcm/foundation/forms/FormStructureHelper.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.day.cq.wcm.foundation.forms;
```
Comment (HitmanInWis) [inline core/src/main/java/com/day/cq/wcm/foundation/forms/FormStructureHelper.java:1]: @id-keenan are all of these classes copied verbatim from the day versions?

File: core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/form/ContainerImpl.java (hunk 1 above)
Comment (brandonmaynard) [reply core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/form/ContainerImpl.java:133]: Restored this functionality. `ExporterConstants.SLING_MODEL_EXTENSION` does not exist in 6.2 so I checked what its value is from https://helpx.adobe.com/experience-manager/6-4/sites/developing/using/reference-materials/javadoc/constant-values.html and hardcoded `"json"`

**Comment by HitmanInWis** [comment]
Closing this out, as all comments have been addressed, and leaving `backport-6.2` as the main backport branch.

**Pull request closed without merging**